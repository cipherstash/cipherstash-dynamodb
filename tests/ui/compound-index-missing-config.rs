use cipherstash_dynamodb::Encryptable;

#[derive(Encryptable)]
struct User {
    #[cryptonamo(compound = "email#name")]
    #[partition_key]
    email: String,
    #[cryptonamo(compound = "email#name")]
    name: String,
}

fn main() {}
