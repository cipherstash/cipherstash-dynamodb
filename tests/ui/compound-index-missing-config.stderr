error: Compound attribute was specified but no query options were. Specify how this field should be queried with the attribute #[cipherstash(query = <option>, compound = "email#name")]
 --> tests/./ui/compound-index-missing-config.rs:5:42
  |
5 |     #[cipherstash(compound = "email#name")]
  |                                          ^
